Makefile

```diff
@@ -108,6 +108,7 @@ verify: tidy download ## Verify code. Includes dependencies, linting, formatting
 	go generate ./...
 	hack/boilerplate.sh
 	curl https://raw.githubusercontent.com/aws/karpenter-core/main/pkg/apis/crds/karpenter.sh_provisioners.yaml > pkg/apis/crds/karpenter.sh_provisioners.yaml
+	curl https://raw.githubusercontent.com/aws/karpenter-core/main/pkg/apis/crds/karpenter.sh_machines.yaml > pkg/apis/crds/karpenter.sh_machines.yaml
```

Comment:
I think it makes sense in `verify`; though, I've updated it so that it pulls from the specific version of `karpenter-core` that is being used by the code

Reply:
```
cp $(KARPENTER_CORE_DIR)/pkg/apis/crds/* pkg/apis/crds
```